### PR TITLE
Servants are now moved to and restricted to Reebe when the Ark starts; also updates the CC changelog

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
balance: Servants are now teleported to the Ark when it activates, and cannot leave Reebe afterwards.
/:cl:

Apparently someone pushed a lit plasma canister into the portals and then a welder. Because teleporting through then causes sparks, this immediately ignites it when someone teleports there, which is silly. Now servants can't make the crew spawn inhospitable with little effort.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_effects/city_of_cogs_rift.dm [hunk 1]
```diff
@@ -46,7 +46,13 @@
 
 /obj/effect/clockwork/city_of_cogs_rift/proc/beckon(atom/movable/AM)
 	var/turf/T = get_turf(pick(GLOB.city_of_cogs_spawns))
-	if(is_servant_of_ratvar(AM))
+	var/servant_pulled = AM.pulledby && is_servant_of_ratvar(AM.pulledby)
+	if(AM.fingerprintslast)
+		for(var/V in GLOB.player_list)
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_effects/city_of_cogs_rift.dm]: why not loop the list of servants?

Comment (Xhuis) [reply code/game/gamemodes/clock_cult/clock_effects/city_of_cogs_rift.dm]: Because I wrote this code really tired and it never occurred to me.

Comment (RemieRichards) [reply code/game/gamemodes/clock_cult/clock_effects/city_of_cogs_rift.dm]: always the simple reasons

**Comment by Tlaltecuhtli** [comment]
only stuff trown by clockists gets in their side of reebe ?

**Comment by Potato-Masher** [comment]
Well, I was about to complain about the crew being able to fuck over the cult with a locker full of max caps sent straight to the Ark, but seeing the code made me smack myself in the face.

**Comment by Robustin** [comment]
Does bumping into dense objects really give it a "fingerprint"? 

As the inventor of the beautiful tactic, would I get banned if I managed to avoid this mechanic as a cultist?

**Comment by Spaceinaba** [comment]
Well it will be a lesson in reading comprehension!

**Comment by Tlaltecuhtli** [comment]
if i drag a monkey with a bomb into a portal where does it go?

**Comment by Xhuis** [comment]
@Tlaltecuhtli Objects dragged don't come with you during transit, and should be pushed in instead.
@Robustin The tactic itself was exploiting a bug, and the change is in the changelog. Other than that, I'm not invested enough in the goings-on of people who use bugs to win a round to warn them prior.

**Comment by Spaceinaba** [comment]
The changelog pops up when you join after an update I'm pretty sure so if you don't read it that's 100% your fault.

**Comment by Potato-Masher** [comment]
No, there were no errors in reading comprehension here @Spaceinaba .  Reading off the auto changelog makes it seem like ANY object thrown or pushed into a rift will end up at the servant's spawn, due to it not specifying that it has to be servants that do it. 

You might want to make that small adjustment @Xhuis just so more people don't get confused. Not everyone looks at github sadly.

**Comment by Xhuis** [comment]
Oh, I didn't even notice that. Sorry! I fixed the entry.

**Comment by Ratvar** [comment]
Can you TK things into enemy spawn as servant? Does telescience works?Conv belts+power from mining bluespace capsule?

**Comment by Xhuis** [comment]
It works based on last fingerprint; if you touch the item, it'll go to the servant spawn. Telescience has been gone for a long time, and I believe telekinesis works based on throwing (which also is affected by fingerprints.)

**Comment by Ratvar** [comment]
So, if servant last touched a bomb-crewmembers can push it with air (be it canister or flamethrowr) to their spawn zone to punch it into servant spawn zone?
Uh, not telesci, launchpads. Do launchpads even work into-out of reebe?

**Comment by Xhuis** [comment]
That is incredibly convoluted, and even though it'd theoretically work, it doesn't strike me as achievable enough. If it does, I can administer a great deal of wrist slapping and fix it.

**Comment by KorPhaeron** [comment]
Why not just force warp all the cultists back before the portals start?

**Comment by Xhuis** [comment]
I don't currently mind them being on station after the timer starts, but it'd be easy to restrict them. Would you prefer that instead?

**Comment by Robustin** [comment]
That would also solve the Warcult issue of "oh look im suddenly glowing and being lynched with no warning".

**Comment by Rinoka** [comment]
Isn't telescience still in the code? I don't recall if it's been completely sealed off or not.

Also Xhius & Ratvar: You forget that Soarium exists. Also, air pumps could do what Ratvar's proposing with very little effort - if it's always stationed in the same room and wrenched down, you can basically make them suck in air and then push it out.

**Comment by Buggy123** [comment]
MERGE THIS SHIT YOU FUCKHEADS THIS IS STILL BROKEN AS HELL, YOU CAN SPAWNKILL LITERALLY THE ENTIRE STATION BY THROWING A PLASMA CAN AND A LIT WELDER THROUGH A PORTAL.

**Comment by Cyberboss** [comment]
Does it crash the server? Leak db info? admin elevation exploit?

None of the above? Fuck you.

**Comment by KorPhaeron** [comment]
@xhuis

Seems the least exploitable that way yes

**Comment by Xhuis** [comment]
#He'sTheShiz 🙏

**Pull request merged**